analytic_engine/src/instance/serial_executor.rs

```diff
@@ -34,6 +38,25 @@ type ScheduleSyncRef = Arc<ScheduleSync>;
 struct ScheduleSync {
     state: Mutex<FlushState>,
     notifier: Sender<()>,
+    retry_flush_count: AtomicUsize,
```

Comment:
I guess a better variable should be called `contionuous_flush_failure_count`. If so, there is no need to increment it in `should_retry_flush`. And add a method `inc_flush_failure_count` and call it when flush fails, which is symmetrical to the case where flush succeeds.

Reply:
done